.circleci/config.yml

```diff
@@ -133,6 +133,10 @@ references:
         then
             echo "export BUCKET_UPLOAD=$BUCKET_UPLOAD_PARAMETER" >> $BASH_ENV
         fi
+        if [ -n "${BACKWARD_COMPATIBILITY}" ];
+        then
+            echo "export CHECK_BACKWARD=$BACKWARD_COMPATIBILITY" >> $BASH_ENV
+        fi
```

Comment:
No need for that just assign this value to `CHECK_BACKWARD`
This is the only place from which the env variable comes from so there is no fear of rewriting it.